projects/opendal/build.sh

```diff
@@ -25,6 +22,12 @@ targets=(
   fuzz_reader
   fuzz_writer
 )
-for target in "${targets[@]}"; do
-  cp ../target/x86_64-unknown-linux-gnu/release/$target $OUT/
-done
+
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_reader_fs
```

Comment:
I prefer to organize code in service order, for exemaple:

- create fs env
- copy files to fuzz-reader_fs